src/java/com/phenix/pct/CompilationAttributes.java

```diff
@@ -280,6 +281,11 @@ public void setCallbackClass(String callback) {
         this.callback = callback;
     }
 
+    @Override
```

Comment:
Really unusure any other output type will be added in the future. So it's probably better to have a boolean controlling the JSON output behavior.
```public void setJsonOutput(boolean json) { ```

Reply:
I was thinking about this, and in my opinion, we should stick to setOutputType
Bcz in future if there is a requirement to get the output in a different form (say XML), in that case, we are closing the door for extensibility.
In my perspective, the present flag is generic enough to handle those cases

Reply:
Not sure anybody will implement XML, but anyway, let's keep the door open :)
However, please enforce that the value is valid. Currently, if we pass 'josn' (with a typo), we don't have the json output, and no message. Stop the build if the value is not correct